app/src/main/java/com/example/listaelementos/domain/models/Endereco.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.listaelementos.domain.models
+
+data class Endereco(
+    val street: String,
+    val city: String,
+    val neighborhood: String,
+    val zip_code: String
```

Comment:
```suggestion
@SerialName("zip_code")
    val zipCode: String
```

o padrão de variavel no android/kotlin é camelCase, pro deserializador entender que é a val certa pelo nome, é recomendado passar o @SerialName em cima, assim vc indica a val certa e pode manter o padrão, inclusive o padrão nosso na DM é botar o @SerialName mesmo que o nome não mude nada pra val